samples/tab-stage-view/csharp/README.md

```diff
@@ -119,30 +119,38 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
     - In the navigation bar located at the far left in Teams, select the ellipses ●●● and choose your app from the list.
 
 ## Running the sample
+- In the navigation bar located at the far left in Teams, select the ellipses ●●● and choose your app from the list.
 
-- Opening stage view in tab with Adaptive card action.
+**Install App:**
 
-![Card Actions](TabInStageView/Images/cardActions.png)
+![InstallApp](TabInStageView/Images/Install.png)
 
-- Opening Collaborative Stage View. Please refer [Collaborative Stage view](https://review.learn.microsoft.com/en-us/microsoftteams/platform/tabs/tabs-link-unfurling?branch=pr-en-us-7891#collaborative-stage-view) for more details
+**Welcome message with feature explanation and Adaptive Card with actions:**
 
-![Stage View](TabInStageView/Images/CollaborativeStageView.png)
+![Welcome Message](TabInStageView/Images/welcomeAction.png)
 
-- Opening stage view from Adaptive card via deep link.
+**Open the URL in tab stage view:**
+
+![InstallApp](TabInStageView/Images/ClickButton.png)
+
+**Opening Collaborative Stage View**. Please refer [Collaborative Stage view](https://review.learn.microsoft.com/en-us/microsoftteams/platform/tabs/tabs-link-unfurling?branch=pr-en-us-7891#collaborative-stage-view) for more details
+
+![Stage View in tab](TabInStageView/Images/CollaborativeStageView.png)
+
+**Opening stage view from Adaptive card via deep link:**
 
 ![Stage View Deep Link](TabInStageView/Images/viaDeeplink.png)
 
-- Opening the stage view from unfurling link. If you copy and paste a link from https://www.BotFramework.com into the compose message area the link will unfurl.
+![LinkUnfurlingStageView](TabInStageView/Images/LinkUnfurlingStageView.png)
 
-![Link Unfurling Text](TabInStageView/Images/LinkUnfurlingText.png)
+**Opening stage view from unfurling link. If you copy and paste a link from https://www.BotFramework.com into the compose message area the link will unfurl.**
 
-![Link Unfurling Card](TabInStageView/Images/LinkUnfurlingCard.png)
+![LinkUnfurlingText](TabInStageView/Images/LinkUnfurlingText.png)
 
-![Link Unfurling StageView](TabInStageView/Images/LinkUnfurlingStageView.png)
+**Tab with execute deep link action to open stage view:**
```

Comment:
You are showing just button with deep-link but after clicking on it what will happen?

Please check all other screenshots and it's next steps like above.

Reply:
Fixed